### PR TITLE
Node server and Contact us form added

### PR DESCRIPTION
**Pull request by MadhavBahl**

### CODE REVIEW

**Comment by Createdd** [comment]
Hi @MadhavBahlMD! Thanks a lot for the Pull Request! Looks great, but it is not working. Changed password and also tried other usernames. Nothing is working. Has it worked on your machine? Checkout out the error I get: 
image
___
Please also see our [Readme](https://github.com/DevelopersRising/DevelopersRising.github.io) to have a valid workflow here. Please
- rename the pull request to somehting meaningful
- try to keep your commits as useful and concise as possible
- reference the issue you are working on
- we don't merge into the master branch. Only the develop branch. Your branch should be named `feature/addContactForm` or something similar

Thanks a lot for all your effort, we highly appreciate it! :) 

@robeerob  Any additional thoughts? Is it working on your machine?

**Comment by MadhavBahl** [comment]
Have you installed NodeJS?

**Comment by MadhavBahl** [comment]
THere might be some error in mailer script i will check it out right away

**Comment by MadhavBahl** [comment]
Is the home page, i.e., localhost:8080/ working?
Or, nothing at all is working?

**Comment by MadhavBahl** [comment]
@DDCreationStudios I am not able to make a new branch **feature/addContactForm** Can you please create one so that i can request pull in that branch

**Comment by Createdd** [comment]
Hi @MadhavBahlMD! The setup works fine - I can start everything.  I just can't send a mail. As the screenshot indicates there has to be a problem with login or the POST route ;) 

But maybe let's wait for @robeerob  and see if he can send it. :)

**Comment by MadhavBahl** [comment]
I will review my code. Will fix it before tomorrow :)

**Comment by MadhavBahl** [comment]
i found the error, it was a very small typo 😞  i named the same route as two different routes on client side and server side. will fix that ASAP

**Comment by MadhavBahl** [comment]
Done :) just add your own email id and password. at first it might show an error that google blocked sign in attemt, then go to your gmail account and review and unblock/authorize this, thereafter it will work perfect 👍

**Review by MadhavBahl — COMMENTED** [review]
POST /feedback route fixed

**Comment by Createdd** [comment]
hi @MadhavBahlMD ! It works great! I let @robeerob  review it and afterwards clean up the code and then merge.
And reagrding the name of the branch: 
You can simply checkout a new branch with the feature name on your current branch and then open a new pull request. But it's not that important, I just wanted  to remark that we have a naming convention. :) 

@robeerob Works for me now. But I activated the option in our gmail account to let "insecure" apps login the account.

**Comment by MadhavBahl** [comment]
Thanks @DDCreationStudios  :)
I dont know why, but i am not able to create a new branch, please create a new branch and i will change the base from develop to feature/AddServer then 👍

File: serverFiles/mailer.js
Comment (robaxelsen) [inline serverFiles/mailer.js:14]: This results in formatting with indentation in received email:
screenshot from 2017-10-29 14-00-26

Please see if you can fix that issue.

File: serverFiles/mailer.js
Comment (robaxelsen) [inline serverFiles/mailer.js:13]: Is this console.log needed anymore? If not, please remove.

File: serverFiles/mailer.js
Comment (robaxelsen) [inline serverFiles/mailer.js]: Please remove.

File: serverFiles/mailer.js
Comment (robaxelsen) [inline serverFiles/mailer.js]: Since you're using ES6, you can replace the use of var here with const.

File: serverFiles/mailer.js
Comment (robaxelsen) [inline serverFiles/mailer.js]: Since you're using ES6, you can replace the use of var within function scope to let.

**Review by robaxelsen — CHANGES_REQUESTED** [review]
Overall, VERY GOOD job on this :+1: :tada: Thanks a lot for wanting to help us, and for your great work putting together this PR. Please see my small suggestions for changes.

**Comment by robaxelsen** [comment]
@DDCreationStudios Apart from my review comments, works well for me locally!

File: serverFiles/mailer.js
Comment (MadhavBahl) [reply serverFiles/mailer.js:14]: Yes it's fixed using template string 👍

Comment (MadhavBahl) [reply serverFiles/mailer.js:14]: image

Fixed 👍

**Comment by MadhavBahl** [comment]
Thanks for the complement @robeerob  😃 .
The requested changes are being worked on 👍

**Comment by Createdd** [comment]
@robeerob Can you update your review when everything is ok for you? Afterwards I will have a final look and merge :)

**Review by robaxelsen — APPROVED** [review]

**Comment by robaxelsen** [comment]
@DDCreationStudios LGTM :+1:

**Review by Createdd — APPROVED** [review]

**Pull request merged**
